yarn.lock

```diff
@@ -10390,13 +7124,3 @@ yocto-queue@^0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
   integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
-
-yorkie@^2.0.0:
-  version "2.0.0"
-  resolved "https://registry.yarnpkg.com/yorkie/-/yorkie-2.0.0.tgz#92411912d435214e12c51c2ae1093e54b6bb83d9"
-  integrity sha512-jcKpkthap6x63MB4TxwCyuIGkV0oYP/YRyuQU5UO0Yz/E/ZAu+653/uov+phdmO54n6BcvFRyyt0RRrWdN2mpw==
-  dependencies:
-    execa "^0.8.0"
-    is-ci "^1.0.10"
-    normalize-path "^1.0.0"
-    strip-indent "^2.0.0"
```

Comment:
I'm guessing the app is going to be a lot lighter with so many dependencies removed from lockfile.

Reply:
Kind of. A lot of these are related to Node.js built in polyfills that we needed because of the way we imported `msw/node` in our mock code. Since this was a file that eventually got imported in our application, we needed to bundle them, too. But I split that out and could remove all of them. So kind of yes in the sense that no more unnecessary msw stuff is in our bundle and kind of no because it shouldn’t have been in there in the first place. So let’s take this as a yes :D